src/pynxtools/nexus/nexus.py

```diff
@@ -602,7 +611,9 @@ def axis_helper(dim, nxdata, signal, axes, logger):
     for a_item in range(dim):
         ax_list = []
         ax_datasets = nxdata.attrs.get("axes")  # primary axes listed in attribute axes
-        ax_list = get_single_or_multiple_axes(nxdata, ax_datasets, a_item, ax_list)
+        ax_list = get_single_or_multiple_axes(
+            nxdata, ax_datasets, a_item, ax_list, logger
```

Comment:
Probably the logger should be setup somewhere globally, so it doesn't have to passed down to each function. But since this is the case everywhere in this tools, let's go with what is suggested here.

Reply:
I agree and I also noticed that `logger` has not been used everywhere or at least should be a global logger for each module, so that we can call when it is needed. 
We should work in a new PR and also check the compatibility of the `pynxtools` logger to the `nomad` logger.  Though, I believe we can see the pynxtools log massage in `nomad`. 

But this PR is not intended for logger fix.

Reply:
Can you make an issue for this please?